### DISCUSSION TITLE
Room cordinats

### DISCUSSION THREAD

**Discussion by elad1994** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#705] [General]
HI, 

thanks for your awsome card. 
im struggling a bit with the cordinate. I was able to retrive the map into the card. 
Now, my goal is to have the abilty to seperate the map into rooms. But im not sure how to get my cordinats. 
also i want over your templetet and i can see that your are using [X1,Y1,X2,Y2] and also [X1,Y1] in the room part. 

Thanks

**Comment by PiotrMachowski** [comment]
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/318
